### PR TITLE
Use Markdown table syntax for the types table

### PR DESCRIPTION
**Pull request by wilzbach**
If https://github.com/stonemaster/dlang-tour/pull/471 is enabled, we can think about the best way to convert the HTML tables to markdown:
- AFAICT the Vibe.d Markdown parser requires a header (we could fork it?)
- Tweaking the CSS (see the screenshots below, we could also drop the dlang.org CSS here and use the a more modern one like e.g. here on GitHub?)

without `table` CSS class:

image

with `table` CSS class:

image

currently

image

### CODE REVIEW

**Comment by ZombineDev** [comment]
LGTM, feel free to merge when ready.

File: basics/basic-types.md [hunk 1]
```diff
@@ -7,21 +7,21 @@ precision. There is no difference
...
-<table class="table table-hover">
-<tr><td width="250px"><code class="prettyprint">float</code></td> <td>32-bit</td></tr>
-<tr><td><code class="prettyprint">double</code></td> <td>64-bit</td></tr>
-<tr><td><code class="prettyprint">real</code></td> <td>depending on platform, 80-bit on Intel x86 32-bit</td></tr>
-</table>
+| type    | size
+|---------|--------------------------------------------------
+|`float`  | 32-bit
+|`double` | 64-bit
+|`real`   | depending on platform, 80-bit on Intel x86 32-bit
```
Comment (ZombineDev) [inline basics/basic-types.md]: I would say ">= 64-bit", depending on the platform - generally 64-bit, but 80-bit on x86".

Comment (stonemaster) [inline basics/basic-types.md]: +1

**Comment by wilzbach** [comment]
> I would say ">= 64-bit", depending on the platform - generally 64-bit, but 80-bit on x86".

Done.

> LGTM, feel free to merge when ready.

Will do so once Travis passes.

**Comment by wilzbach** [comment]
> I would say ">= 64-bit", depending on the platform - generally 64-bit, but 80-bit on x86".

Short follow-up - I removed "depending on the platform" in order to shorten the table width:

image

shortened:

image

**Pull request merged**